NFluidsynth/Native/LibFluidsynth.Misc.cs

```diff
@@ -6,14 +6,14 @@ namespace NFluidsynth.Native
     internal static partial class LibFluidsynth
     {
         [DllImport(LibraryName)]
-        [return: MarshalAs(UnmanagedType.I4)]
+        [return: MarshalAs(UnmanagedType.I1)]
```

Comment:
Hrm. This *shouldn't* be correct as the Fluidsynth source code definitely says it's of type `int`.

I would prefer you made it `int` instead and made usages check instead.

Reply:
Yeah, but for some reason it crashed on my computer, complaining about it being I4...
I've changed it to return int now